OnlineShopUI/online-shop-ui/src/components/Login.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import Modal from '@mui/material/Modal';
+import CloseIcon from '@mui/icons-material/Close';
+import './Login.scss';
+import FormGroup from '@mui/material/FormGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Checkbox from '@mui/material/Checkbox';
+import User from '../img/icon-user.png';
+
+export default function Login() {
+  const [open, setOpen] = React.useState(false);
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+
+  return (
+    <div>
+      <a href="#">
+        <img src={User} alt="user" className="log-btn" onClick={handleOpen} />
+      </a>
+      <Modal
+        className="log-modal"
+        open={open}
+        onClose={handleClose}
+        aria-labelledby="modal-modal-title"
+        aria-describedby="modal-modal-description"
+      >
+        <Box>
+          <div className="login">
+            <div className="login__type">
+              <p>
+                <strong>Вход</strong>
+              </p>
+              <CloseIcon
+                className="login__btnclose"
+                onClick={handleClose}
+              ></CloseIcon>
+            </div>
+            <div className="login__date">
+              <div className="login__email">
+                <label>Эл. почта или телефон</label>
+                <input type="text" placeholder="example@gmail.com" />
+              </div>
+              <div className="login__pass">
+                <label>Пароль</label>
+                {/* <i>
+                <LockIcon className="lock"></LockIcon>
+              </i> */}
+                <input type="text" placeholder="1d$5D03" />
```

Comment:
Why input for password is of type 'text' and why we need a placeholder for password?
It's better to provide logic of password over the input like 'a-z, A-Z, 0-9, #